components/delivery-option-messages.js

```diff
@@ -0,0 +1,153 @@
+// Delivery Frequency codes
+const ONLY_WEEKEND_DELIVERY_FREQ = 'NWE';
+const FIVE_DAYS_WEEK_DELIVERY_FREQ = 'N5D';
+const SIX_DAYS_WEEK_DELIVERY_FREQ = 'N6D';
+
+// Distributor Type codes
+const HAND_DELIVERY = 'HD';
+const MAIL = 'ML';
+
+// Country codes
+const USA_COUNTRY_CODE = 'USA';
+const CAN_COUNTRY_CODE = 'CAN';
+
+/**
+ * This is a static list of messages for delivery options. The items are created by the combination of some distributor properties:
+ * - deliveryFrequency (NWE, N5D, N6D) - is defined based on the selected product.
```

Comment:
I get why you changed that and we push you that way, but looking at the requirements now, it may be best to keep it as:
```suggestion
 * - deliveryFrequency (A1, A5, A6) - is defined based on the selected product (codes containing NWE, N5D, N6D).
```
So that it makes sense when we go back on tickets and look into salesforce and other data sources.